EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+#### Start sniffing
+
+There are three ways to start sniffing:
+* If this is your first time using an interface, click on interface options 
+  to set channel and baudrate, then click start. Check the IEEE 802.15.4 tap 
+  to ensure that the channel information is included in the pcap output and 
+  can be displayed in the Wireshark GUI. The parameters will be saved after 
+  the start of the capture, and you will not need to set it again the next 
+  time you use the interface (unless you need to change the channel)
+
+* Double click on the hardware interface
+* Select the hardware interface by clicking on it and then click the Wireshark 
+  icon on the top left to start sniffing.
+ 
+ #### Capture from multiple hardware interfaces/boards
+ Select all hardware interfaces in the Capture Screen in Wireshark and click 
+ the Wireshark icon on the top left to start sniffing.
+ 
+**Interface ID (frame.interface_id)**\
+Interface Identifier used by Wireshark to identify the capture interfaces
+
+**Interface name (frame.interface_name)**\
+Interface Identifier used by Wireshark to identify the capture interfaces 
+
+**Channel (wpan-tap.ch_num)**
+
+## Troubleshooting
+### The OT sniffer is not listed in the Wireshark interface. 
+1. If you have multiple Python interpreters installed, ensure that Python2 interpreter is used by the extcap script. The sniffer.py doesn't support python3 yet.
+2. See if the hardware has been enumerated on USB and the drivers are loaded. 
+3. Check that the HEX file for the hardware has been flashed. 
+4. Reset the hardware by unplugging the hardware, waiting 5 seconds, and plugging it back in. 
+5. Restart Wireshark. If it still doesn’t appear, verify the python script located in the extcap folder is able to run. 
+    
+    For OS X and Linux:
+    1. Verify that the execute permission is present for the extcap_ot.py file. 
+        ```
+       ls -l extcap_ot.py
+        ```
+    2. If the "x" permission is missing:
+       ```
+       chmod +x extcap_ot.py
+       ```
+    3. Run ```python extcap_ot.py --extcap-interfaces``` to list the interface
+    
+    For Windows:
+    1. Run ```extcap_ot.bat --extcap-interfaces``` to list the interface.
+    2. If this exits with a python error, verify that python.exe can be run from the command line ```C:>python.exe --version```
+#### Wireshark only allow the root user to capture packets
+During the Wireshark installation on Ubuntu the user will be prompted to choose one of the following options:
+* Create the wireshark user group and allow all members of that group to capture packets.
+* Only allow the root user to capture packets.
+
+**Note**: Using the Wireshark as the root user is strongly discouraged.
+
+To change the settings after the installation, run the following command:
+```
+sudo dpkg-reconfigure wireshark-common
+```
+If the Wireshark was configured to restrict the capture to members of the wireshark group, add the correct user to the group:
```

Comment:
`wireshark` group

use code font to differentiate it from the Wireshark product